apps/modernization-ui/src/apps/patient/add/extended/inputs/administrative/Administrative.tsx

```diff
@@ -0,0 +1,56 @@
+import { Controller, useFormContext } from 'react-hook-form';
+import { AdministrativeEntry } from 'apps/patient/data/entry';
+import { Input } from 'components/FormInputs/Input';
+import { maxLengthRule } from 'validation/entry';
+import { DatePickerInput } from 'components/FormInputs/DatePickerInput';
+import { Wrapper } from '../../wrapper/Wrapper';
+
+export const Administrative = () => {
+    const { control } = useFormContext<{ administrative: AdministrativeEntry }>();
+
+    return (
+        <Wrapper title="Administrative" id="administrative">
```

Comment:
issue: i think the id should be something like "section-Administrative" so the right hand side nav can link to the right section. Check AddPatientExtendedNav.tsx for the exact id value